go/private/context.bzl

```diff
@@ -45,14 +45,20 @@ INFERRED_PATH = "inferred"
 
 EXPORT_PATH = "export"
 
-def _declare_file(go, path="", ext="", name = ""):
-  filename = mode_string(go.mode) + "/"
-  filename += name if name else go._ctx.label.name
+def _declare_child(go, path, ext, name):
```

Comment:
Nit: `_child_name` or `_child_path` might be a better name for this function. It doesn't actually declare anything.